src/transaction/CustomFeeLimit.js

```diff
@@ -0,0 +1,121 @@
+import AccountId from "../account/AccountId.js";
+import CustomFixedFee from "../token/CustomFixedFee.js";
+
+/**
+ * @namespace proto
+ * @typedef {import("@hashgraph/proto").proto.IFixedFee} HashgraphProto.proto.IFixedFee
+ * @typedef {import("@hashgraph/proto").proto.ICustomFeeLimit} HashgraphProto.proto.ICustomFeeLimit
+ */
+
+export default class CustomFeeLimit {
+    /**
+     *
+     * @param {object} props
+     * @param {?AccountId | string} [props.accountId]
+     * @param {?CustomFixedFee[]} [props.fees]
```

Comment:
same as the previous comment. I think `CustomFixrdFee[]`without the `?` should be enough